sdk/identity/azure-identity/azure/identity/_credentials/azure_cli.py

```diff
@@ -67,7 +71,8 @@ def get_token(self, *scopes, **kwargs): # pylint: disable=no-self-use
 
         resource = _scopes_to_resource(*scopes)
         command = COMMAND_LINE.format(resource)
-        tenant = resolve_tenant("", **kwargs)
+        tenant = resolve_tenant(default_tenant= "", **kwargs) if self.tenant_id is None else resolve_tenant(default_tenant= "", tenant_id= self.tenant_id, **kwargs)
```

Comment:
Given the definition of resolve_tenant is:

def resolve_tenant(default_tenant, tenant_id=None, **_):

why not just use 

tenant = resolve_tenant(default_tenant= "", tenant_id= self.tenant_id, **kwargs)

Reply:
With the last update I did where I was explicit about the params there isn't a reason not to. It does make it similar to read and I have updated it.